test/forms/inputs/input.test.js

```diff
@@ -9,43 +9,43 @@ const input = { name, value, onChange }
 
 test('Input defaults to text input', () => {
   const props = { input, meta: {} }
-  const wrapper = mount(<Input {...props} />)
-  expect(wrapper.find('input').prop('type')).toEqual('text')
+  render(<Input {...props} />)
+  expect(screen.getByLabelText('Field')).toHaveAttribute('type', 'text')
 })
 
 test('Input contains div with class input-wrapper', () => {
   const props = { input, meta: {} }
-  const wrapper = mount(<Input {...props} />)
-  expect(wrapper.find('div.input-wrapper').exists()).toEqual(true)
+  render(<Input {...props} />)
+  expect(screen.getByLabelText('Field').parentElement).toHaveClass('input-wrapper')
 })
 
 test('Input renders children', () => {
   const Wrapped = () => <p> I'm a child component </p>
   const props = { input, meta: {} }
-  const wrapper = mount(
+  render(
     <Input {...props}>
       <Wrapped />
     </Input>
   )
-  expect(wrapper.find(Wrapped).exists()).toEqual(true)
+  expect(screen.getByText("I'm a child component")).toBeInTheDocument()
 })
 
 test('Input is given an aria-describedby attribute when there is an input error', () => {
   const props = { input, meta: { touched: true, invalid: true } }
-  const wrapper = mount(<Input {...props} />)
-  expect(wrapper.find('input').prop('aria-describedby')).toContain(name)
+  render(<Input {...props} />)
+  expect(screen.getByLabelText('Field').getAttribute('aria-describedby')).toContain(name)
```

Comment:
We've been using .toHaveAttribute('aria-describedby', name) in other places.

Reply:
There's a subtle distinction here. The error attribute uses a custom `generateInputErrorId` util, which appends `Error` to the field name. So `.toHaveAttribute('aria-describedby', name)` would fail because it's actually `.toHaveAttribute('aria-describedby', `${name}Error`)`.

However, I wanted to try and decouple it a bit from the implementation of `generateInputErrorId`. So if for example we changed the implementation to `return name + __error`, this input test **wouldn't** fail. But maybe I'm trying to be too clever?

Reply:
No, that makes sense. I didn't notice we were using `toContain` instead of `toEqual` here.